riposte-spi/src/test/java/com/nike/riposte/server/http/impl/RequestInfoImplTest.java

```diff
@@ -433,6 +433,7 @@ public void getContent_returns_null_if_request_is_not_complete_with_all_chunks()
             "/some/path/foo/bar/ | /some/path/{param1}/{param2}  | foo | bar",
             "/some/path/foo/bar  | /some/path/{param1}/{param2}/ | foo | bar",
             "/some/path/foo/bar/ | /some/path/{param1}/{param2}/ | foo | bar",
+            "/some/path/%E5%A5%BD/%E6%B5%8B%E8%AF%95/ | /some/path/{param1}/{param2}/ | %E5%A5%BD | %E6%B5%8B%E8%AF%95",
```

Comment:
Whoops, I should have checked how this test was working before making the suggestion in the issue. To fix this test to actually test what we want, change the first part to this:

```
// given
RequestInfo<TestContentObject> requestInfo = new RequestInfoImpl<>(path, null, null, null, null, null, null, null, null, false, true, false);

// when
```

The rest of the test can stay the same, but then you'd change `expectedParam1` and `expectedParam2` to be the decoded versions.